x/poolmanager/create_pool.go

```diff
@@ -86,11 +86,6 @@ func (k Keeper) CreatePool(ctx sdk.Context, msg types.CreatePoolMsg) (uint64, er
 	}
 
 	emitCreatePoolEvents(ctx, poolId, msg)
-	// TODO: Remove gamm hooks and only use poolmanager hook for all types of poolType
-	// only call the hook through poolmanager if the pool created is CL
-	if msg.GetPoolType() == types.Concentrated {
-		k.poolCreationListeners.AfterPoolCreated(ctx, sender, pool.GetId())
-	}
```

Comment:
Note: this is now run in `InitializePool` in `x/concentrated-liquidity`. Thoroughly tested by a follow-up PR https://github.com/osmosis-labs/osmosis/pull/4831